File: C & C++ PRACTICE CODES/INVERTED TRIANGLE PATTERN/code4.c
#include <stdio.h>


void main(){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0 ; i < row ; i++){

		for(int j = 0 ; j < row - i ; j++){

			if((row-j)%2 == 1){

				printf("%d\t",row - j);

			}else{

				printf("%c\t",64+row-j);
			}
		}

		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/9/code4.java

import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s= new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();
		int temp = 1;

		
		for(int i = 1; i <= row ; i++){
		

			for(int j = 0; j < row ; j++){

				if(i % 2 == 0){

					System.out.print(((temp+j)*2)+"		");
				}else{
					System.out.print((j+i)+"		");

				}
			}
			if(i%2==0){

				temp++;
			}

				System.out.print("\n\n");

		}

	}

}
